### DISCUSSION TITLE
Github repos

### DISCUSSION THREAD

**Discussion by luisa289803** [community/community#46651] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Are there guidelines for naming repositories?

**Comment by airtower-luna** [comment, marked as answer]
I'd say it's common to stick to lower case letters, numbers, and dashes, but it's not a hard rule. Usually the name of the project works well, maybe add a component name if you have multiple repositories for different parts of one project.

**Comment by Juanescacha** [comment]
They're are not guidelines and you can use any name you like (you decide) as long as its available (the green checkmark)
![image](https://user-images.githubusercontent.com/6394715/218266067-8d7d5f98-e0d4-48e3-9df5-61e8599ea6a6.png)

But besides that, as you been told, its suggested to use lower case letters, with combination of Dashes or camelCase style for having a good naming to YOUR projects, but it's a matter of your preference only.

**Comment by lbirkert** [comment]
I see a lot of people applying the naming conventions from their respected languages the project uses on the repository name. For example with java it is UpperCamelCase. Use this as a rule of thumb but keep in mind that you can really name your project anything you like, especially lowercase with dashes is really common, as it resembles a domain name.

**Comment by luisa289803** [comment]
Thank you all!

**Reply by Juanescacha** [reply]
???
